tests/framework-tests/preferences/js/PanelsTests.js

```diff
@@ -1286,6 +1321,58 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         }]
     });
 
+
+    fluid.defaults("fluid.tests.textSizePanelJQuery", {
+        gradeNames: ["fluid.test.testEnvironment"],
+        components: {
+            textSize: {
+                type: "fluid.tests.prefs.panel.textSizeJQuery",
+                container: ".flc-textSize-jQuery",
+                createOnEvent: "{textSizePanelJQuery}.events.onSwitchedToJqueryWidgets"
+            },
+            textSizeTester: {
+                type: "fluid.tests.textSizeJQueryTester"
+            }
+        },
+        events: {
+            "onSwitchedToJqueryWidgets": null
+        }
+    });
+
+    fluid.defaults("fluid.tests.textSizeJQueryTester", {
+        gradeNames: ["fluid.test.testCaseHolder"],
+        testOptions: {
+            newValue: 1.2
+        },
+        modules: [{
+            name: "Test the text sizer settings panel (jQuery slider)",
+            tests: [{
+                expect: 4,
+                name: "Test the rendering of the text size panel",
+                sequence: [{
+                    func: "fluid.tests.switchToJqueryWidgets",
+                    args: ["{textSizePanelJQuery}"]
+                },  {
```

Comment:
It seems like a lot of the test sequences are duplicated. I think you factor these out so that the sequences are stored as an array and passed in here. The difference I noticed between them is with the first element in the sequence which sets which context to use. I think you could rewrite the switch function to something like:

```javascript
fluid.tests.switchWidgetTo = function (widgetType, testPanel) {
    fluid.contextAware.makeChecks({
        "fluid.prefsWidgetType": {
            value: widgetType
        }
    });

    testPanel.events.onSwitchedWidget.fire();
};
```